impl/src/main/java/io/perfmark/impl/SecretPerfMarkImpl.java

```diff
@@ -308,6 +316,25 @@ protected void attachTag(String tagName, String tagValue) {
       }
     }
 
+    static <T> void handleTaskNameFailure(
+        T taskNameObject, StringFunction<? super T> stringFunction, Throwable cause) {
+      try {
+        if (logger.isLoggable(Level.WARNING)) {
+          LogRecord lr =
+              new LogRecord(
+                  Level.WARNING, "PerfMark.startTask ignored: taskObject={0}, stringFunction={1}");
```

Comment:
Looks like this string is out-of-date. The startTask was not ignored.

Reply:
Oops, I'll fix it tonight.  Nice catch.